src/components/Sidebar/components/CloseButton/CloseButton.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import styled, { css } from 'react-emotion';
+import PropTypes from 'prop-types';
+
+import CloseIcon from './closeIcon.svg';
+
+const baseStyles = ({ theme, visible }) => css`
+  label: close-button;
+  outline: none;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 40px;
+  width: 40px;
+  border-radius: 50%;
+  background-color: ${theme.colors.n100};
+  position: absolute;
+  bottom: ${theme.spacings.mega};
+  right: ${theme.spacings.mega};
+  transition: opacity 150ms ease-in-out, visibility 150ms ease-in-out;
+  visibility: ${visible ? 'visible' : 'hidden'};
+  opacity: ${visible ? 1 : 0};
+  z-index: 1;
+  ${theme.mq.mega`
+    visibility: hidden;  
+  `};
+`;
+
+const FloatingButton = styled('button')(baseStyles);
+
+const CloseButton = ({ visible, onClick }) => (
+  <FloatingButton visible={visible} onClick={onClick}>
+    <CloseIcon />
```

Comment:
The SVG isn't accessible to screenreaders either since it lacks a `<title>` element or `aria-label` attribute. However, in this case, it's better to explicitly hide the icon from screen readers since it's purely for presentational purposes and a descriptive label is enough to convey the function of the button. Therefore, please add the `aria-hidden="true"` attribute to the icon. 

```suggestion
    <CloseIcon aria-hidden="true" />
```